CONTRIBUTING.md

```diff
@@ -13,19 +14,16 @@ and feel free to propose changes to this document in a pull request.
 ### Useful docs contributing links
 
 * [I just have a quick question](https://wiki.hyperledger.org/display/BESU/I+just+have+a+quick+question)
-* [How to Contribute]
-* [Reporting Bugs in the Doc](https://wiki.hyperledger.org/display/BESU/Reporting+Bugs+in+the+Doc)
+* [How to contribute](https://wiki.hyperledger.org/display/BESU/How+to+Contribute)
+* [Reporting bugs in the documentation](https://wiki.hyperledger.org/display/BESU/Reporting+Bugs+in+the+Doc)
 * [Submitting a Documentation Enhancement Suggestion](https://wiki.hyperledger.org/display/BESU/Suggesting+Documentation+Enhancements)
 * [Documentation Style Guide](https://wiki.hyperledger.org/display/BESU/Besu+Documentation+Style+Guide)
 * [Pull Request Labels](https://wiki.hyperledger.org/display/BESU/Pull+Request+Labels)
-
 * [Security](SECURITY.md)
 
-
 ### Other important information
 
-* [Besu Roadmap](https://wiki.hyperledger.org/display/BESU/Roadmap)
+* [Besu roadmap](https://wiki.hyperledger.org/display/BESU/Roadmap)
```

Comment:
Do other titles like "Code of Conduct", "Submitting a Documentation Enhancement Suggestion" , "Documentation Style Guide" and "Pull Request Labels" have to be sentence cased too?

Reply:
I would have thought so if that is our style.

Reply:
Updated, plus fixed up the long links.